Musigwa/RNLevelUp
# #166258383 Design the Application's Home Screen

#### What does this PR do?

- It finishes the home screen
#### Description of Task to be completed?

- Set up the application's state management [React Redux]
- Fetch the list of Java developers based in Lagos

#### How should this be manually tested?
- After cloning the repository, check out `ft-design-homescreen` branch
- Checkout the `README.md` file for set up instructions
- Run `npm install` to install the project's dependencies
- Run `npm test` to runs all test scenarios and see the results in from the terminal
- After all, run `npm start` to open the application in `ios simulation` or `android emulator` or just a physical device. 
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #166258383
#### Screenshots (if appropriate)
- Android device

<img width="331" alt="Screen Shot 2019-06-07 at 10 34 15" src="https://user-images.githubusercontent.com/29999424/59091535-d99d7300-890f-11e9-8a95-4180bca7b1bb.png">

- IOS device

<img width="349" alt="Screen Shot 2019-06-07 at 10 37 03" src="https://user-images.githubusercontent.com/29999424/59091730-41ec5480-8910-11e9-85f2-b4fdbe15e64d.png">

#### Questions:
- None
